paimon-spark/paimon-spark-common/src/test/java/org/apache/paimon/spark/SparkFilterConverterTest.java

```diff
@@ -192,6 +210,11 @@ public void testDate() {
 
     @Test
     public void testIgnoreFailure() {
+        testStartsWith();
```

Comment:
No need modify this, this is only to test `convertIgnoreFailure`, regardless of the filter used

Reply:
OK，removed the ut